flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
Any specific reason it's not being built from source?

Reply:
I had originally configured things expecting to build a flatpak in our CI and then deploy the existing artifact elsewhere in our existing release automation.
When I realised I'd have to configure a different system to build the package, this is the adaptation. I got a contributor that figured out most things and collecting our built executable avoids having to figure out how to install Python 3.11 in the Flatpak.

As for x86_64 only as well, I've recently had to help an arm64 macOS user and we found out one of our dependencies is a rust project that seems to fail to build on arm64. So that's a blocker right now.